emle/models.py

```diff
@@ -47,13 +47,26 @@
 
 try:
     import NNPOps as _NNPOps
-
+    import NNPOps.neighbors.getNeighborPairs as _getNeighborPairs
     _NNPOps.OptimizedTorchANI = _torchani_patches.OptimizedTorchANI
 
     _has_nnpops = True
-except:
+except ImportError:
```

Comment:
Please leave this as a general exception so that we catch _everything_ not just `ImportError`. I know that this is the likely exception, but I'd rather that the module works regardless of what went wrong, e.g. if there's a bug in the module, or one of it's dependencies, then this wouldn't necessarily be raised as an `ImportError`.